CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file.
 
+## v1.3.TODO - 2023-01-11
+
+### Added 
+- option to receive notification when switching from a work project without notes
+- confirmation on closing KeepTime 
+- import functionality of database
+- icons for dialogs
+- Collapse / Expand button for the project report 
+
+### Changed
+- updated the Settings design
+- replaced deprecated fontawesome dependencies
+- updated dependencies 
+- isWork is now the default when creating a new project
+- autofocus the note field instead of the project name when editing a project
+- active work item is not editable anymore
+- simplify note format
+
+### Fixed 
+- "hide projects" on linux 
+- notes disappear after an auto safe 
+- creating a project without a name
```

Comment:
```suggestion
- option to receive note reminder only when switching from a work project without notes
- option to show confirmation on closing KeepTime 
- import functionality of database (to be used with an export of the export functionality)
- app icon shown in dialogs
- collapse / expand button for the project report 

### Changed
- updated the settings dialog design
- replaced deprecated fontawesome dependencies
- updated dependencies - we now require Java 11 (instead of 8)
- when creating a new project 'isWork' is selected by default
- when editing a note the note field will be in focus instead of the project
- in the report dialog the currently active work item is not editable anymore (marked as 'active work' instead)
- in report dialog simplified the note format when copying them (removed the note count)
- when creating/editing a project the name cannot be empty anymore

### Fixed 
- "hide projects" setting was not working correctly on linux. we disabled the setting for linux.
```